Automattic/wp-calypso
# Jetpack checklist: Inline tasks

#### Changes proposed in this Pull Request

* Move tasks that were spread across two files into the checklist.
* Move duration function into component
* Fix `complete` type error due to irregular checklist state
* Drop `flowRight` for easier typing
* Add `LocalizeProps`

#### Testing instructions

* Ensure the Jetpack Checklist works as expected (no functional changes).
* Different plan levels will include different checklist tasks, so try a few different plans.
* Visit `/plans/my-plan/SITE_SLUG` for a Jetpack site

Closes #33950
